apiVersion: events.k8s.io/v1
deprecatedCount: 1
deprecatedFirstTimestamp: "2023-03-09T14:32:59Z"
deprecatedLastTimestamp: "2023-03-09T14:32:59Z"
deprecatedSource:
  component: kubelet
  host: vrutkovs-sno
eventTime: null
kind: Event
metadata:
  creationTimestamp: "2023-03-09T14:33:01Z"
  managedFields:
  - apiVersion: v1
    fieldsType: FieldsV1
    fieldsV1:
      f:count: {}
      f:firstTimestamp: {}
      f:involvedObject: {}
      f:lastTimestamp: {}
      f:message: {}
      f:reason: {}
      f:source:
        f:component: {}
        f:host: {}
      f:type: {}
    manager: kubelet
    operation: Update
    time: "2023-03-09T14:33:01Z"
  name: oauth-openshift-7749b8f584-djk22.174ac6c97fb8737d
  namespace: openshift-authentication
  resourceVersion: "16201"
  uid: b69640f5-fc75-4c1d-aedb-2391e0b2d62a
note: "Readiness probe error: Get \"https://10.128.0.74:6443/healthz\": dial tcp 10.128.0.74:6443:
  connect: connection refused\nbody: \n"
reason: ProbeError
regarding:
  apiVersion: v1
  fieldPath: spec.containers{oauth-openshift}
  kind: Pod
  name: oauth-openshift-7749b8f584-djk22
  namespace: openshift-authentication
  resourceVersion: "15138"
  uid: 6f6eb656-5372-4dd2-958f-29163b5fec6a
type: Warning
